fastq_filtrator.py

```diff
@@ -0,0 +1,241 @@
+# Here is fastq filtrator
+
+
+# GC-filtering
+def filter_gc(input_list, gc_bounds, save_filtered):
+    input_list_1 = list()

```

Comment:
`output_list_passed` больше бы подошёл по смыслу
```suggestion
    output_list_passed = list()

```